### ISSUE TITLE
HLS Support

### ISSUE THREAD

**Issue by dxpsite** [dularion/streama#537]
Can you add support HLS in "External URL" option, because our content placed on Amazon S3 throught Flussonic Media Server?
For example: **https://someurl/private/s01e02.the.cap.table-1.mp4/index.m3u8**
Thanx!

**Comment by Shatatel** [comment]
Upvote for flussonic URLs like:
http:/server.name:80/path/file-name.mp4/index.m3u8
http:/server.name:80/path/file-name.mp4/video.m3u8
http:/server.name:80/path/file-name.mp4/mono.m3u8
http:/server.name:80/path/file-name.mp4/Manifest.mpd

**Comment by dxpsite** [comment]
no. only link link like http:/server.name:80/path/file-name.mp4/ may be use.
m3u8 recognized as not html5 format

**Comment by brixxnatt** [comment]
can we add live stream channels ??? but link is m3u8

**Comment by umairali4433** [comment]
my s3 bucket is public but still i cant connect that bucket in flussonic anyone can help?
